uftrace.c

```diff
@@ -712,6 +722,7 @@ int main(int argc, char *argv[])
 		.color		= COLOR_AUTO,  /* default to 'auto' (turn on if terminal) */
 		.column_offset	= 8,
 		.comment	= true,
+		.recv_cmd	= NULL,
```

Comment:
In general, omitted fields will be initialized to zero so no need to do it explicitly.